### ISSUE TITLE
Icon for Lincity-NG

### ISSUE THREAD

**Issue by hlechner** [numixproject/numix-icon-theme-circle#1928]
Game: **Lincity-NG**
Description: **A city simulation game**
Website: https://sourceforge.net/projects/lincity-ng.berlios/
Icon:
![image](https://cloud.githubusercontent.com/assets/11244067/7097718/af92daa0-dfae-11e4-81e0-e7f988aa1d80.png)

note: It's hardcoded (due to **.png** in **Icon=** line)

lincity-ng.desktop:
```
[Desktop Entry]
Type=Application
Version=1.0
Encoding=UTF-8
Name=Lincity-NG
GenericName=City Simulation Game
GenericName[de]=Stadtsimulationsspiel
Comment=Plan and Build a city and take care of transport, economics, electricityand others.
Icon=lincity-ng.png
Exec=lincity-ng
Terminal=false
StartupNotify=false
Categories=Game;Simulation;
```

**Comment by Foggalong** [comment]
For a game with respectable sprite graphics, it sure does have a shit icon and UI.

**Comment by Foggalong** [comment]
Fixed in 6c74225f14d82c659180286a502e529e30f588fd
